src/renderer/library/components/searchResult/AllPublicationPage.tsx

```diff
@@ -397,6 +397,17 @@ const CellGlobalFilter: React.FC<ITableCellProps_GlobalFilter> = (props) => {
     // className={classNames(classThemeExample)}
     // className={classNames(classStyleExample)}
 
+    const location = useLocation();
+    const queryParams = new URLSearchParams(location.search);
+    const searchValue = queryParams.get("value") || "";
+
+    React.useEffect(() => {
+        if (props.focusInputRef?.current) {
+            props.focusInputRef.current.value = searchValue;
+            onInputChange(searchValue || undefined);
+        }
+    }, [searchValue, onInputChange, props.focusInputRef]);
```

Comment:
in the dependency array, I think `props.focusInputRef` should be `props.focusInputRef?.current?.value` because the React "ref" is likely stable but its "current" value is what changes.